packages/client/src/pages/Login/login.scss

```diff
@@ -1,53 +1,167 @@
-    display: flex;
-    justify-content: center;
-
-    .title {
-        text-align: center;
-        font-family: $default-font-family;
+    & * {
+        box-sizing: border-box;
+        transition: all 0.25s ease;
     }
 
-    .login-content {
-        width: 20rem;
-        display: flex;
-        flex-direction: column;
-
-        .input {
-            font-family: $default-font-family;
-            font-size: $input-font-size;
-            border: 1px solid $silver-gray;
-            padding-left: 10px;
-            padding-right: 10px;
-            margin-bottom: 0.5rem;
-            margin-top: 0rem;
-            border-radius: 8px;
-
-            &::placeholder {
-                color: $bright-gray;
+    height: 40rem;
+    width: 100%;
+    padding: 60px 0;
+    background-size: cover;
+    background-image: linear-gradient(rgb(255, 255, 255) 0%, rgba(255, 255, 255, 0) 30%),
+        url('boat-4899802_1920.jpg');
+
+    .login-window {
+        height: 35rem;
+        max-width: 30rem;
+        padding: 2rem;
+        margin: 0 auto;
+        background-color: rgba(255, 255, 255, 0.9);
+        border-radius: 0.5rem;
+        overflow: hidden;
+
+        .login-nav {
+            width: 100%;
+
+            ul {
+                height: 2rem;
+                padding: 0;
+                margin: 0 0 2rem 0;
             }
-        }
 
-        input {
-            height: 2.5rem;
-        }
+            li {
+                display: inline;
+                padding: 0 0.6rem 0.3rem 0.6rem;
+                font-size: $subtitle-font-size;
+                text-align: left;
+                text-transform: uppercase;
+                color: rgba(13, 59, 104, 0.9);
+                text-decoration: none;
+                cursor: pointer;
+                @include animated-underline($powder-blue);
+                &:after {
+                    height: 2.5px;
+                }
 
-        .login-buttons {
-            margin-top: 1rem;
+                &.active {
+                    &:after {
+                        width: 100%;
+                    }
+                }
 
-            .button {
-                margin-bottom: 0.3rem;
+                &.nav-right {
+                    float: right;
+                }
+            }
+        }
+
+        .form {
+            form {
                 width: 100%;
-                justify-content: center;
+                height: 27rem;
+                color: $tc-black;
+                font-size: $default-font-size;
+
+                label {
+                    display: block;
+                    padding-left: 0.625rem;
+                    margin-bottom: 0.625rem;
+                    text-transform: uppercase;
+                }
+
+                input {
+                    display: block;
+                    width: 100%;
+                    height: 2.5rem;
+                    padding: 0 2rem 0 0.625rem;
+                    background: rgba(255, 255, 255, 0);
```

Comment:
```suggestion
                    background: rgba($tc-white, 0);
```

Reply:
Dangg, I didn't realize scss can convert hex to rgba for us, that is so useful haha